integration/test/basic/basic_test.go

```diff
@@ -31,7 +31,7 @@ func TestHelm(t *testing.T) {
 }
 
 func installCertManager(ctx context.Context, helmClient helmclient.Interface, logger micrologger.Logger) error {
-	tarballURL, err := appcatalog.GetLatestChart(ctx, catalogURL, certManagerAppName, "")
+	tarballURL, err := appcatalog.GetLatestChart(ctx, catalogURL, certManagerAppName, "0.9.0")
```

Comment:
we need to set this for now as the latest `cert-manager` has completely new CRD API and there is no easy migration path, we need to think about how we can solve this in the next release ( we cannot solve in this because we still need to upgrade to `0.9.0` before we can skip to the new version